js/src/api/rpc/parity/parity.js

```diff
@@ -95,6 +95,11 @@ export default class Parity {
       .execute('parity_dappsList');
   }
 
+  dappsRefresh () {
```

Comment:
I don't think the RPC method (and JS code) will be useful at all if the router can auto-refresh missing dapps.